### PR TITLE
Azure Stack Update Admin Specification

### PR DESCRIPTION
**Pull request by deathly809**
This is for Azure Stack **ONLY**.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
@deathly809 Have these APIs been reviewed by ARM and the API review board?

File: specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/Updates.json [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+              }
+          },
+          "description": "Returns a list of support REST operations.",
+          "tags": [
+              "Update"
+          ],
+          "operationId": "Operations_List",
+          "parameters": [
+              {
+                  "$ref": "#/parameters/ApiVersionParameter"
```
Comment (jhendrixMSFT) [inline specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/Updates.json]: This ref needs to be fixed, see failure [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/364293753).

File: specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateRuns.json [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+      "post": {
+        "x-ms-examples": {
+          "Rerun an update.": {
+            "$ref": "./examples/UpdateRuns/Rerun.json"
+          }
+        },
+        "tags": [
+          "UpdateRuns"
+        ],
+        "description": "Get a specific update version.",
```
Comment (jhendrixMSFT) [inline specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateRuns.json]: Is this description accurate?

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Also have you had a look at the model validation failures [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/364293763).

**Comment by deathly809** [comment]
@jhendrixMSFT Yeah, the guy that worked on this did some "stuff".  I am fixing it now, will have it fixed by tomorrow.  I have no clue about any of the descriptions, we sent out the files to the teams, both PMs and devs and everyone said "looks great!" ha ha.....  Message me to hear how I really feel.

File: specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateRuns.json (hunk 2 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateRuns.json]: The update team signed off on the spec after we asked them to review and make sure the descriptions are okay.  Guess they missed this.  I have changed it to something that is more correct.

**Comment by deathly809** [comment]
@jhendrixMSFT  I have fixed the spec and examples.  This will never go into Azure as it is only for Azure Stack.

File: specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/Updates.json (hunk 1 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/Updates.json]: Done.

**Comment by jhendrixMSFT** [comment]
@deathly809 The spacing in some of the docs is wrong, they should all use two space chars for indentation.  I think this is the only remaining part to clean up.  Regarding your previous comment about this not going into Azure, does this mean that the ARM team doesn't need to review the APIs?  We still need to build SDKs for this right?

**Comment by deathly809** [comment]
@jhendrixMSFT Can you show me where the 2 indent requirement is please?  I really don't like it being that small because it is hard for me to read.  This is part of the Azure Stack administration process and will not be a part of Azure.

**Comment by jhendrixMSFT** [comment]
Hmm I don't know if this is a formal requirement.  I bring it up because there have been talks about tooling to enforce this (or maybe do it for you).  @fearthecowboy do you have anything to add?

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**